.github/workflows/docker.yml

```diff
@@ -0,0 +1,145 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "master"
+
+jobs:
+  # Builds base image for stempy of various verions if Dockerfile.base has changed
+  build-base:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - version: stempy
+            tag-suffix: -conda-jammy
+          - version: stempy-mpi
+            tag-suffix: -conda
+          - version: stempy-mpi
+            tag-suffix: -conda-jammy
+
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      -
+        name: Determine if Dockerfile.base changed
+        id: changed-dockerfile-base
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            docker/${{ matrix.version }}${{ matrix.tag-suffix}}/Dockerfile.base
+      -
+        name: Login to Docker Hub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+      -
+        name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      -
+        name: Build/push Dockerfile.base
+        uses: docker/build-push-action@v3
+        if: ${{ contains(github.event.head_commit.message, 'trigger-ci') || steps.changed-dockerfile-base.outputs.any_changed == 'true'}}
+        with:
+          context: ./docker/${{ matrix.version }}${{ matrix.tag-suffix}}
+          file: ./docker/${{ matrix.version }}${{ matrix.tag-suffix}}/Dockerfile.base
+          push: true
+          tags: ${{ secrets.DOCKERHUB_USERNAME }}/${{ matrix.version }}-base:latest${{ matrix.tag-suffix }}
```

Comment:
We will be pushing the images to the "openchemistry" repo which will be different to `DOCKERHUB_USERNAME`, so you an either hardcode it here or we can setup another variable.

Reply:
Changed to `${{ vars.DOCKHUB_ORG }}`